dev/com.ibm.ws.ejbcontainer.core/src/com/ibm/ws/metadata/ejb/EJBMDOrchestrator.java

```diff
@@ -5011,7 +5015,10 @@ else if (!bmd.metadataComplete) {
 
             // F743-13022 - Check now if the runtime environment supports timers
             // rather than waiting until the automatic timers are created.
-            bmd.container.getEJBRuntime().setupTimers(bmd);
+            if (isTraceOn && tc.isDebugEnabled())
+                Tr.debug(tc, "Are persistent timers present? " + hasPersistentTimers);
+            if (hasPersistentTimers)
```

Comment:
Note: if persistent timers exist in the database, they won't run if the server is restarted and there are no `@Schedule` persistent timers.